internal/service/opensearch/domain_test.go

```diff
@@ -1493,6 +1545,54 @@ func TestAccOpenSearchDomain_VolumeType_update(t *testing.T) {
 		}})
 }
 
+// Verifies that EBS volume_type can be changed from gp3 to a type which does not
+// support the throughput and iops input values (ex. gp2)
+//
+// Reference: https://github.com/hashicorp/terraform-provider-aws/issues/27467
+func TestAccOpenSearchDomain_VolumeType_GP3ToGP2(t *testing.T) {
```

Comment:
```suggestion
func TestAccOpenSearchDomain_VolumeType_gp3ToGP2(t *testing.T) {
```
Nit: The final tuple/clause/thing should be lowercase in Go.